include/vpi_user.h

```diff
@@ -853,6 +853,8 @@ typedef struct t_cb_data
 #define cbNBASynch               30
 #define cbAtEndOfSimTime         31
 
+/**************************** Verilog AMS ****************************/
+#define vpiRealNet             526
```

Comment:
Where does this number come from?

Reply:
``include/vpi_user.h`` is otherwise pretty much unchanged from the same file in my Cadence Xcelium installation. It might be wise to keep it unchanged and do it like Cadence do, i.e. provide extra include files specific to a simulator version (like the ``vpi_user_cds.h`` where that define came from).

Reply:
`vpi_user.h` is taken directly from the Verilog-2005 standard, Annex G (http://staff.ustc.edu.cn/~songch/download/IEEE.1364-2005.pdf). Instead of modifying it, please create a new file with these extensions, similar to `include/sv_vpi_user.h`.
And please add documentation where this extension comes from/what tools are using it. It's hard to forget where magic numbers are coming from as time passes.